### PR TITLE
Implement feedback

### PR DESCRIPTION
**Pull request by patrex**
#### What does this PR do?
- adds heroku Procfile
- changes filenames
- changes export style
- updates package.json test script
- adds coveralls for test coverage
- fixes homepage
- adds styles(background images) to signIn.html, signUp.html and others

#### How should this be manually tested?
 ```javascript
    npm test
```
#### What are the relevant pivotal tracker stories?
#165411998

### CODE REVIEW

**Comment by kampkelly** [comment]
Your pull request description is lacking some information. Also, some of your commit messages do not have a body and footer.

File: UI/js/adminDashboard.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+  fetch(url)
+    .then(user => user.json())
+    .then((user) => {
+      imageURI.src = user.results[0].picture.large;
+      username.innerHTML = `${user.results[0].name.first} ${(user.results[0].name.last)}`
+    })
+    .catch((err) => {
+    });
+}
```
Comment (kampkelly) [inline UI/js/adminDashboard.js:29]: add new line

**Comment by kampkelly** [comment]
Good work.
Please look into the following:
Give your class names like 
```
export default class AccountsValidator {
	…
}
```
Also please use full names for variable names. For instance in transactionsRouter.js, `import as TransactionsController`, etc.
Also, there is a comment on usersController.js, line 18. Please remove it.

**Comment by kampkelly** [comment]
Can you update me on the request to add Joi errors together so the user sees all at once?

**Comment by kampkelly** [comment]
Your branch name is incorrect, please look at the document I sent earlier.

**Comment by patrex** [comment]
> Can you update me on the request to add Joi errors together so the user sees all at once?

Turns out there's an option ```abortEarly``` that controls this behavior. I'm going to try to see how it works

**Pull request closed without merging**